bigtable/client/table.cc

```diff
@@ -194,5 +194,35 @@ bool Table::CheckAndMutateRow(std::string row_key, Filter filter,
   return response.predicate_matched();
 }
 
+Row Table::CallReadModifyWriteRowRequest(
+    btproto::ReadModifyWriteRowRequest row_request) {
+  auto error_message =
+      "ReadModifyWriteRowRequest(" + row_request.table_name() + ")";
+  auto response_row = RpcUtils::CallWithoutRetry(
+      *client_, rpc_retry_policy_->clone(), metadata_update_policy_,
+      &StubType::ReadModifyWriteRow, row_request, error_message.c_str());
+
+  std::vector<bigtable::Cell> cells;
+  for (auto& family : response_row.row().families()) {
```

Comment:
I believe you want:

```C++
  auto& row = *response_row.mutable_row();
  for (auto& family : *row.mutable_families()) {
    for (auto& column : *family.mutable_columns()) {
      for (auto cell : *column.mutable_cells()) {
```

Otherwise you are still making copies of the data.